projects/js-packages/jetpack-cli/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "automattic/jetpack-cli",
+	"description": "Development tools for the Jetpack monorepo",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"test-js": [
+			"echo 'no tests yet'"
+		]
```

Comment:
Probably no tests ever, since installing `jest` won't be possible with it being excluded from `pnpm-workspace.yaml`.

In which case, again, deleting this so CI will be able to skip it entirely might be good.